environment_setup/setup_software.sh

```diff
@@ -157,16 +157,14 @@ sudo cp "$CURR_DIR/../src/software/autoref/DIV_B.txt" "/opt/tbotspython/autoRefe
 
 print_status_msg "Finished setting up AutoRef"
 
-# Install Bazel
-print_status_msg "Installing Bazel"
+# Install Bazelisk. The first time bazel command is ran, the appropriate bazel version will be downloaded
+print_status_msg "Installing Bazelisk"
 
-# Adapted from https://docs.bazel.build/versions/main/install-ubuntu.html#install-with-installer-ubuntu
-sudo wget -nc https://github.com/bazelbuild/bazel/releases/download/5.0.0/bazel-5.0.0-installer-linux-x86_64.sh -O /tmp/bazel-installer.sh
-sudo chmod +x /tmp/bazel-installer.sh
-sudo /tmp/bazel-installer.sh --bin=/usr/bin --base=$HOME/.bazel
-echo "source ${HOME}/.bazel/bin/bazel-complete.bash" >> ~/.bashrc
```

Comment:
I wasn't sure if we needed to configure the `--base` flag and how that can be done

`bazel-complete` also seems to not be properly supported with Bazelisk installation as of now (https://github.com/bazelbuild/bazelisk/issues/29), and given that we use the `./tbots.py` script anyways, I've opted to ignore that for now.